src/main.cpp

```diff
@@ -327,12 +359,17 @@ int main(int argc, char *argv[])
 		// Get VRAM usage
 		float vramUsage = getVramUsage(nvmlLibrary);
 
+		// Check that we're in a supported application
+		bool inDashboard = vr::VROverlay()->IsDashboardVisible();
+		bool isCurrentAppDisabled = disabledApps.find(getCurrentApplicationKey()) != disabledApps.end();
+		bool adjustResolution = !inDashboard && !isCurrentAppDisabled;
```

Comment:
This should probably be inside the if (line 368) so that we don't call the OpenVR API too often. Can make adjustResolution global so that line 499-502 can still work.

Reply:
Fixed in d74ca41.